docs/userguides/platform.md

```diff
@@ -20,6 +20,23 @@ The Platform UI will let you create and manage Clusters using a graphical experi
 The CLI experience is for those working locally who don't want to visit the website, or are locally developing their applications.
 ```
 
+Once you have created your Cluster, you have to fund it so it is made available for your use.
+To do that, use the [`silverback cluster pay create`][silverback-cluster-pay-create] command to fund your newly created cluster.
+Please note that provisioning your cluster will take time, and it may take up to an hour for it to be ready.
```

Comment:
suggestion: I think it's safe to lower that expectation a bit to no more than 30 minutes. I can't seem to find my initial reference for some reason, but I thought the AWS docs mentioned up to 15 minutes. And in my experience it hasn't taken that long.  

Did you see longer or something?

Reply:
Just leaving space in case we have something slowing it down. In practice I see just under 10 minutes.

But also, the ApePay stream cannot be touched until after 1 hour, which is sort of the hard limit for safety purposes (not allow more changes to the stream parameters while infra changes are occuring)

We can move these parameters inwards later

Reply:
Just seems like we're shouting at the user that "this will be so incredibly slow you'll grow a beard" before they even get to try it. Though at least they'll be pleasantly surprised with a 10 minutes deploy. :smile:

Reply:
I think the opposite, if you don't explicitly call out that it is going to take longer than like ~5 seconds or so, then it might be confusing why you can't use it immediately, and it's great to share that expectation early

This seems more like a UX expectation thing, so I'll let @joshcoppola give his opinion

Reply:
I agree with @fubuloubu that it's better to be honest that things will take some time rather than leave a user confused that they can't instantly use the cluster. 

How about something like "Provisioning a cluster usually takes 10 minutes, but could take as long as an hour"? That communicates the expected time while still giving some buffer if things do take way longer than we expect. 

Then we could also follow up with something like "For safety purposes, ApePay streams are always locked for one hour" to communicate that side of things (I'm out of my element on how to phrase that but you get the idea)

Reply:
I didn't mean to imply you were advocating for lying, I meant to agree with @fubuloubu 's point about sharing the time expectation with users.

I don't have an informed opinion on exactly how much buffer time would be good to communicate, mainly I wanted to point out that we could include the actual expected time and another value with some buffer time in the message.

Even though it is separate from cluster deployment, I think it's worth mentioning the ApePay cancellation window somewhere if we don't already call that out somewhere else.

Reply:
I think we can fuse it all together and say the expectation is 10 minutes, but it may take longer, and you can't cancel it until an hour has passed.